annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -153,6 +157,7 @@ protected void validateTranscriptionWithFullTextResource(Body body)
     }
 
     private void validateFullTextResource(Body body) throws PropertyValidationException {
+    if(body==null) return;
```

Comment:
if the argument is null, that is an error case that cannot be ignored. To avoid NPE, a PropertyValidationException must be thrown

Reply:
Improved.